Flaky test_dont_move_sync_msgs

https://github.com/chatmail/core/actions/runs/20645510691/job/59284343405?pr=7669

```
___________________________ test_dont_move_sync_msgs ___________________________
[gw3] linux -- Python 3.14.2 /home/runner/work/core/core/python/.tox/py/bin/python

acfactory = <deltachat.testplugin.ACFactory object at 0x7fbb2afb7c50>

    def test_dont_move_sync_msgs(acfactory):
        ac1 = acfactory.new_online_configuring_account(bcc_self=True, sync_msgs=True, fix_is_chatmail=True)
        acfactory.bring_accounts_online()
    
        ac1.direct_imap.select_folder("Inbox")
        # Sync messages may also be sent during the configuration.
        inbox_msg_cnt = len(ac1.direct_imap.get_all_messages())
    
        ac1.set_config("displayname", "Alice")
        ac1._evtracker.get_matching("DC_EVENT_MSG_DELIVERED")
        ac1.set_config("displayname", "Bob")
        ac1._evtracker.get_matching("DC_EVENT_MSG_DELIVERED")
        ac1.direct_imap.select_folder("Inbox")
>       assert len(ac1.direct_imap.get_all_messages()) == inbox_msg_cnt + 2
E       assert 2 == (1 + 2)
E        +  where 2 = len([<imap_tools.message.MailMessage object at 0x7fbb1e3bbd90>, <imap_tools.message.MailMessage object at 0x7fbb1e3bbed0>])
E        +    where [<imap_tools.message.MailMessage object at 0x7fbb1e3bbd90>, <imap_tools.message.MailMessage object at 0x7fbb1e3bbed0>] = get_all_messages()
E        +      where get_all_messages = <deltachat.direct_imap.DirectImap object at 0x7fbb2aee0180>.get_all_messages
E        +        where <deltachat.direct_imap.DirectImap object at 0x7fbb2aee0180> = <Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_dont_move_sync_msgs0/ac1/dc.db>.direct_imap

tests/test_1_online.py:284: AssertionError
----------------------------- Captured stdout call -----------------------------
[acsetup] 0.045 started configure on <Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_dont_move_sync_msgs0/ac1/dc.db>
bringing accounts online
wait_all_configured finds accounts= {<Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_dont_move_sync_msgs0/ac1/dc.db>: 'CONFIGURING'}
2.16 [events-ac1] INFO src/scheduler.rs:69: starting IO
2.16 [events-ac1] INFO src/scheduler.rs:393: Starting inbox loop.
2.16 [events-ac1] INFO src/scheduler.rs:412: Preparing new IMAP session for inbox.
2.16 [events-ac1] INFO src/imap.rs:341: Connecting to IMAP server.
2.16 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2.16 [events-ac1] INFO src/scheduler.rs:693: Starting simple loop for Mvbox.
ac1 waiting for inbox IDLE to become ready
2.16 [events-ac1] INFO src/scheduler.rs:758: Starting SMTP loop.
2.16 [events-ac1] INFO src/scheduler.rs:716: Preparing new IMAP session for Mvbox.
2.16 [events-ac1] INFO src/imap.rs:341: Connecting to IMAP server.
2.16 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2.16 [events-ac1] INFO src/imap.rs:356: IMAP trying to connect to ci-chatmail.testrun.org:993:tls.
2.16 [events-ac1] INFO src/net/dns.rs:134: Using memory-cached DNS resolution for ci-chatmail.testrun.org.
2.16 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 49.12.217.82.
2.16 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 2a01:4f8:c013:13c3::1.
2.16 [events-ac1] INFO src/scheduler.rs:935: scheduler is running
2.16 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2.16 [events-ac1] INFO src/contact.rs:2101: Recently seen loop waiting for 24h 0m 0s or interrupt
2.16 [events-ac1] INFO src/location.rs:730: Location loop is waiting for 24h 0m 0s or interrupt
2.16 [events-ac1] INFO src/ephemeral.rs:609: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2.16 [events-ac1] INFO src/imap/client.rs:122: Attempting IMAP connection to ci-chatmail.testrun.org (49.12.217.82:993).
2.16 [events-ac1] INFO src/imap.rs:356: IMAP trying to connect to ci-chatmail.testrun.org:993:tls.
2.16 [events-ac1] INFO src/net/dns.rs:134: Using memory-cached DNS resolution for ci-chatmail.testrun.org.
2.16 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 49.12.217.82.
2.16 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 2a01:4f8:c013:13c3::1.
2.16 [events-ac1] INFO src/ephemeral.rs:609: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
2.16 [events-ac1] INFO src/imap/client.rs:122: Attempting IMAP connection to ci-chatmail.testrun.org (49.12.217.82:993).
2.16 [events-ac1] INFO src/smtp.rs:513: Selected rows from SMTP queue: [1].
2.16 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
2.16 [events-ac1] INFO src/smtp.rs:133: SMTP trying to connect to ci-chatmail.testrun.org:465:tls.
2.16 [events-ac1] INFO src/net/dns.rs:134: Using memory-cached DNS resolution for ci-chatmail.testrun.org.
2.16 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 49.12.217.82.
2.16 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 2a01:4f8:c013:13c3::1.
2.16 [events-ac1] INFO src/smtp/connect.rs:108: Attempting SMTP connection to ci-chatmail.testrun.org (49.12.217.82:465).
2.45 [events-ac1] INFO src/imap.rs:394: Logging into IMAP server with LOGIN.
2.45 [events-ac1] INFO src/imap.rs:394: Logging into IMAP server with LOGIN.
2.46 [events-ac1] INFO src/smtp/connect.rs:108: Attempting SMTP connection to ci-chatmail.testrun.org ([2a01:4f8:c013:13c3::1]:465).
2.46 [events-ac1] WARNING src/smtp/connect.rs:137: Failed to connect to ci-chatmail.testrun.org ([2a01:4f8:c013:13c3::1]:465): connection failure: Network is unreachable (os error 101).
2.81 [events-ac1] DC_EVENT_SMTP_CONNECTED data1=0 data2=SMTP-LOGIN as ci-c228e7@ci-chatmail.testrun.org ok
2.81 [events-ac1] INFO src/smtp.rs:388: Try number 1 to send message Msg#12 (entry 1) over SMTP.
2.81 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
3.11 [events-ac1] INFO src/imap.rs:404: Enabling IMAP compression.
3.17 [events-ac1] INFO src/smtp/send.rs:58: Message len=3178 was SMTP-sent to ci-c228e7@ci-chatmail.testrun.org.
3.17 [events-ac1] DC_EVENT_SMTP_MESSAGE_SENT data1=0 data2=Message len=3178 was SMTP-sent to ci-c228e7@ci-chatmail.testrun.org
3.17 [events-ac1] DC_EVENT_MSG_DELIVERED data1=10 data2=12
3.17 [events-ac1] DC_EVENT_CHATLIST_ITEM_CHANGED data1=10 data2=0
3.17 [events-ac1] INFO src/smtp.rs:615: Sending MDNs.
3.17 [events-ac1] INFO src/scheduler.rs:798: SMTP fake idle started.
3.17 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
3.17 [events-ac1] INFO src/scheduler.rs:826: SMTP has no messages to retry, waiting for interrupt.
3.17 [events-ac1] INFO src/scheduler.rs:830: SMTP fake idle interrupted.
3.17 [events-ac1] INFO src/smtp.rs:513: Selected rows from SMTP queue: [].
3.17 [events-ac1] INFO src/smtp.rs:615: Sending MDNs.
3.17 [events-ac1] INFO src/scheduler.rs:798: SMTP fake idle started.
3.17 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
3.17 [events-ac1] INFO src/scheduler.rs:826: SMTP has no messages to retry, waiting for interrupt.
3.20 [events-ac1] INFO src/imap.rs:404: Enabling IMAP compression.
3.21 [events-ac1] DC_EVENT_IMAP_CONNECTED data1=0 data2=IMAP-LOGIN as ci-c228e7@ci-chatmail.testrun.org
3.21 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
3.21 [events-ac1] INFO src/imap.rs:437: Successfully logged into IMAP server.
3.30 [events-ac1] DC_EVENT_IMAP_CONNECTED data1=0 data2=IMAP-LOGIN as ci-c228e7@ci-chatmail.testrun.org
3.30 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
3.30 [events-ac1] INFO src/imap.rs:437: Successfully logged into IMAP server.
3.31 [events-ac1] INFO src/imap.rs:1826: Scanning folder: Name { owner: ResponseData { raw: 4096, response: MailboxData(List { name_attributes: [Extension("\\HasNoChildren")], delimiter: Some("."), name: "INBOX" }) }, dependent: InnerName { attributes: [Extension("\\HasNoChildren")], delimiter: Some("."), name: "INBOX" } }
3.31 [events-ac1] INFO src/imap.rs:1852: Using "." as folder-delimiter.
3.31 [events-ac1] INFO src/imap.rs:1768: Looking for MVBOX-folder "DeltaChat"...
3.40 [events-ac1] INFO src/imap.rs:1826: Scanning folder: Name { owner: ResponseData { raw: 4096, response: MailboxData(List { name_attributes: [Extension("\\HasNoChildren")], delimiter: Some("."), name: "INBOX" }) }, dependent: InnerName { attributes: [Extension("\\HasNoChildren")], delimiter: Some("."), name: "INBOX" } }
3.40 [events-ac1] INFO src/imap.rs:1852: Using "." as folder-delimiter.
3.40 [events-ac1] INFO src/imap.rs:1768: Looking for MVBOX-folder "DeltaChat"...
3.42 [events-ac1] INFO src/imap.rs:1768: Looking for MVBOX-folder "INBOX.DeltaChat"...
3.50 [events-ac1] INFO src/imap.rs:1768: Looking for MVBOX-folder "INBOX.DeltaChat"...
3.62 [events-ac1] INFO src/imap/select_folder.rs:102: Failed to select folder "DeltaChat" because it does not exist, trying to create it.
3.69 [events-ac1] INFO src/imap/select_folder.rs:102: Failed to select folder "DeltaChat" because it does not exist, trying to create it.
3.79 [events-ac1] INFO src/imap/select_folder.rs:105: Couldn't select folder, then create() failed: no response: code: None, info: Some("[ALREADYEXISTS] Mailbox already exists (0.001 + 0.000 secs).").
3.84 [events-ac1] INFO src/imap/select_folder.rs:80: Selected folder "DeltaChat".
3.84 [events-ac1] INFO src/imap/select_folder.rs:258: transport 1: UID validity for folder DeltaChat changed from 0/0 to 1767304812/1.
3.84 [events-ac1] INFO src/imap.rs:1798: MVBOX-folder DeltaChat created.
3.84 [events-ac1] INFO src/imap.rs:1864: Setting MVBOX FOLDER TO DeltaChat
3.84 [events-ac1] INFO src/imap.rs:1880: FINISHED configuring IMAP-folders.
3.89 [events-ac1] INFO src/imap/select_folder.rs:80: Selected folder "DeltaChat".
3.89 [events-ac1] INFO src/imap.rs:1798: MVBOX-folder DeltaChat created.
3.89 [events-ac1] INFO src/imap.rs:1864: Setting MVBOX FOLDER TO DeltaChat
3.89 [events-ac1] INFO src/imap.rs:1880: FINISHED configuring IMAP-folders.
3.90 [events-ac1] INFO src/imap.rs:578: No new emails in folder "DeltaChat".
3.99 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
3.99 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
3.99 [MAIN-ac1] inbox IDLE ready
finished, account2state {<Account path=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_dont_move_sync_msgs0/ac1/dc.db>: 'IDLEREADY'}
all accounts online
3.99 [events-ac1] INFO src/scheduler.rs:669: IMAP session in folder "DeltaChat" supports IDLE, using it.
4.04 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
4.04 [events-ac1] INFO src/sql.rs:942: Start housekeeping...
4.04 [events-ac1] INFO src/sql.rs:1003: 2 files in use.
4.04 [events-ac1] INFO src/sql.rs:819: Incremental vacuum freed 0 pages.
4.05 [events-ac1] INFO src/sql.rs:724: wal_checkpoint: Total time: 4.073719ms. Writers blocked for: 510.369µs. Readers blocked for: 482.137µs.
4.05 [events-ac1] INFO src/sql.rs:923: Housekeeping done.
4.09 [events-ac1] INFO src/imap/idle.rs:64: IDLE entering wait-on-remote state in folder "DeltaChat".
4.25 [events-ac1] INFO src/imap/select_folder.rs:80: Selected folder "INBOX".
4.25 [events-ac1] INFO src/imap/select_folder.rs:258: transport 1: UID validity for folder INBOX changed from 0/0 to 1767304811/3.
4.29 [events-ac1] DC_EVENT_ACCOUNTS_ITEM_CHANGED data1=0 data2=0
4.29 [events-ac1] DC_EVENT_CONFIG_SYNCED data1=0 data2=0
4.29 [events-ac1] INFO src/scheduler.rs:830: SMTP fake idle interrupted.
4.29 [events-ac1] INFO src/ephemeral.rs:609: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
4.29 [events-ac1] INFO src/smtp.rs:513: Selected rows from SMTP queue: [2].
4.29 [events-ac1] INFO src/smtp.rs:388: Try number 1 to send message Msg#13 (entry 2) over SMTP.
4.29 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
4.45 [events-ac1] INFO src/imap.rs:831: Done fetching existing messages.
4.45 [events-ac1] INFO src/imap.rs:1558: Server supports metadata, retrieving server comment and admin contact.
4.54 [events-ac1] INFO src/net/dns.rs:134: Using memory-cached DNS resolution for ci-chatmail.testrun.org.
4.54 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 49.12.217.82.
4.54 [events-ac1] INFO src/net/dns.rs:180: Resolved ci-chatmail.testrun.org into 2a01:4f8:c013:13c3::1.
4.54 [events-ac1] INFO src/imap.rs:606: fetch_new_msg_batch(INBOX): UIDVALIDITY=1767304811, UIDNEXT=3.
4.62 [events-ac1] INFO src/smtp/send.rs:58: Message len=3146 was SMTP-sent to ci-c228e7@ci-chatmail.testrun.org.
4.62 [events-ac1] DC_EVENT_SMTP_MESSAGE_SENT data1=0 data2=Message len=3146 was SMTP-sent to ci-c228e7@ci-chatmail.testrun.org
4.62 [events-ac1] DC_EVENT_MSG_DELIVERED data1=10 data2=13
4.62 [events-ac1] DC_EVENT_CHATLIST_ITEM_CHANGED data1=10 data2=0
4.62 [events-ac1] INFO src/smtp.rs:615: Sending MDNs.
4.62 [events-ac1] INFO src/scheduler.rs:798: SMTP fake idle started.
4.62 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
4.62 [events-ac1] INFO src/scheduler.rs:826: SMTP has no messages to retry, waiting for interrupt.
4.62 [events-ac1] INFO src/scheduler.rs:830: SMTP fake idle interrupted.
4.62 [events-ac1] INFO src/smtp.rs:513: Selected rows from SMTP queue: [].
4.62 [events-ac1] INFO src/smtp.rs:615: Sending MDNs.
4.62 [events-ac1] INFO src/scheduler.rs:798: SMTP fake idle started.
4.62 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
4.62 [events-ac1] INFO src/scheduler.rs:826: SMTP has no messages to retry, waiting for interrupt.
4.62 [events-ac1] DC_EVENT_ACCOUNTS_ITEM_CHANGED data1=0 data2=0
4.62 [events-ac1] DC_EVENT_CONFIG_SYNCED data1=0 data2=0
4.62 [events-ac1] INFO src/scheduler.rs:830: SMTP fake idle interrupted.
4.62 [events-ac1] INFO src/ephemeral.rs:609: Ephemeral loop waiting for deletion in 24h 0m 0s or interrupt
4.62 [events-ac1] INFO src/smtp.rs:513: Selected rows from SMTP queue: [3].
4.62 [events-ac1] INFO src/smtp.rs:388: Try number 1 to send message Msg#14 (entry 3) over SMTP.
4.62 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
4.64 [events-ac1] INFO src/imap.rs:799: 0 mails read from "INBOX".
4.64 [events-ac1] INFO src/imap/scan_folders.rs:39: Starting full folder scan
4.74 [events-ac1] INFO src/imap/scan_folders.rs:98: Found folders: ["DeltaChat", "INBOX"].
4.74 [events-ac1] INFO src/imap.rs:578: No new emails in folder "INBOX".
4.84 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
4.84 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
4.84 [events-ac1] INFO src/scheduler.rs:669: IMAP session in folder "INBOX" supports IDLE, using it.
4.84 [events-ac1] INFO src/imap.rs:1915: Need to refetch "INBOX", got unsolicited FETCH Other(ResponseData { raw: 4096, response: Fetch(1, [ModSeq(7), Flags(["\\Seen"])]) })
4.84 [events-ac1] INFO src/imap.rs:1904: Need to refetch "INBOX", got unsolicited EXISTS Exists(2)
4.84 [events-ac1] INFO src/imap/idle.rs:39: Skipping IDLE in "INBOX" because there may be new mail.
4.84 [events-ac1] INFO src/scheduler.rs:425: IMAP loop iteration for inbox finished, keeping the session.
4.84 [events-ac1] INFO src/imap.rs:606: fetch_new_msg_batch(INBOX): UIDVALIDITY=1767304811, UIDNEXT=3.
4.94 [events-ac1] INFO src/imap.rs:799: 1 mails read from "INBOX".
5.04 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
5.04 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
5.04 [events-ac1] INFO src/scheduler.rs:669: IMAP session in folder "INBOX" supports IDLE, using it.
5.04 [events-ac1] INFO src/imap/idle.rs:47: Skip IDLE in "INBOX" because we got interrupt.
5.04 [events-ac1] INFO src/scheduler.rs:425: IMAP loop iteration for inbox finished, keeping the session.
5.05 [events-ac1] INFO src/smtp/send.rs:58: Message len=3134 was SMTP-sent to ci-c228e7@ci-chatmail.testrun.org.
5.05 [events-ac1] DC_EVENT_SMTP_MESSAGE_SENT data1=0 data2=Message len=3134 was SMTP-sent to ci-c228e7@ci-chatmail.testrun.org
5.05 [events-ac1] DC_EVENT_MSG_DELIVERED data1=10 data2=14
5.05 [events-ac1] DC_EVENT_CHATLIST_ITEM_CHANGED data1=10 data2=0
5.05 [events-ac1] INFO src/smtp.rs:615: Sending MDNs.
5.05 [events-ac1] INFO src/scheduler.rs:798: SMTP fake idle started.
5.05 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
5.05 [events-ac1] INFO src/scheduler.rs:826: SMTP has no messages to retry, waiting for interrupt.
5.05 [events-ac1] INFO src/scheduler.rs:830: SMTP fake idle interrupted.
5.05 [events-ac1] INFO src/smtp.rs:513: Selected rows from SMTP queue: [].
5.05 [events-ac1] INFO src/smtp.rs:615: Sending MDNs.
5.05 [events-ac1] INFO src/scheduler.rs:798: SMTP fake idle started.
5.05 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
5.05 [events-ac1] INFO src/scheduler.rs:826: SMTP has no messages to retry, waiting for interrupt.
5.14 [events-ac1] INFO src/imap.rs:1152: Marked messages 3 in folder INBOX as seen.
5.14 [events-ac1] INFO src/imap.rs:578: No new emails in folder "INBOX".
5.23 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
5.23 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
5.23 [events-ac1] INFO src/scheduler.rs:669: IMAP session in folder "INBOX" supports IDLE, using it.
5.23 [events-ac1] INFO src/imap.rs:1904: Need to refetch "INBOX", got unsolicited EXISTS Exists(3)
5.23 [events-ac1] INFO src/imap/idle.rs:39: Skipping IDLE in "INBOX" because there may be new mail.
5.23 [events-ac1] INFO src/scheduler.rs:425: IMAP loop iteration for inbox finished, keeping the session.
5.23 [events-ac1] INFO src/imap.rs:606: fetch_new_msg_batch(INBOX): UIDVALIDITY=1767304811, UIDNEXT=4.
5.33 [events-ac1] INFO src/imap.rs:799: 1 mails read from "INBOX".
5.43 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
5.43 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
5.43 [events-ac1] INFO src/scheduler.rs:669: IMAP session in folder "INBOX" supports IDLE, using it.
5.43 [events-ac1] INFO src/imap/idle.rs:47: Skip IDLE in "INBOX" because we got interrupt.
5.43 [events-ac1] INFO src/scheduler.rs:425: IMAP loop iteration for inbox finished, keeping the session.
--------------------------- Captured stdout teardown ---------------------------
5.53 [events-ac1] INFO src/imap.rs:1152: Marked messages 4 in folder INBOX as seen.
5.53 [events-ac1] INFO src/imap.rs:578: No new emails in folder "INBOX".
5.63 [events-ac1] DC_EVENT_CONNECTIVITY_CHANGED data1=0 data2=0
5.63 [events-ac1] DC_EVENT_IMAP_INBOX_IDLE data1=0 data2=None
5.63 [events-ac1] INFO src/scheduler.rs:669: IMAP session in folder "INBOX" supports IDLE, using it.
5.73 [events-ac1] INFO src/imap/idle.rs:64: IDLE entering wait-on-remote state in folder "INBOX".
=============== Bob ===============
ARCH=64 AUTHSERV_ID_CANDIDATES= BCC_SELF=1 
BLOBDIR=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_dont_move_sync_msgs0/ac1/dc.db-blobs 
BOT=0 CONFIGURED_INBOX_FOLDER=INBOX CONFIGURED_MVBOX_FOLDER=DeltaChat 
CONFIGURED_TRASH_FOLDER=<unset> 
DATABASE_DIR=/tmp/pytest-of-runner/pytest-0/popen-gw3/test_dont_move_sync_msgs0/ac1/dc.db 
DATABASE_ENCRYPTED=false DATABASE_VERSION=144 
DEBUG_ASSERTIONS=On - DO NOT RELEASE THIS BUILD DEBUG_LOGGING=0 
DELETE_DEVICE_AFTER=0 DELETE_SERVER_AFTER=0 DELETE_TO_TRASH=<unset> 
DELTACHAT_CORE_VERSION=v2.35.0 DISABLE_IDLE=false 
DONATION_REQUEST_NEXT_CHECK=1769896828 DOWNLOAD_LIMIT=0 
FAIL_ON_RECEIVING_FULL_MSG= FETCHED_EXISTING_MSGS=true 
FINGERPRINT=2E6FA2CB23B532D728634B5864B08F61A9ED9443 FIRST_KEY_CONTACTS_MSG_ID= 
FIX_IS_CHATMAIL=true FOLDERS_CONFIGURED=5 GOSSIP_PERIOD=172800 
IMAP_SERVER_ADMIN="mailto:root@ci-chatmail.testrun.org" 
IMAP_SERVER_COMMENT="Chatmail server" IMAP_SERVER_ID={"name": "Dovecot"} 
IS_CHATMAIL=false IS_MUTED=false JOURNAL_MODE=wal 
LAST_CANT_DECRYPT_OUTGOING_MSGS=0 LAST_HOUSEKEEPING=1767304830 LAST_MSG_ID=0 
LEVEL=awesome MDNS_ENABLED=1 MEDIA_QUALITY=0 MESSAGES_IN_CONTACT_REQUESTS=0 
MVBOX_MOVE=0 NUM_CPUS=4 NUMBER_OF_CHAT_MESSAGES=5 NUMBER_OF_CHATS=2 
NUMBER_OF_CONTACTS=0 ONLY_FETCH_MVBOX=0 PRIVATE_KEY_COUNT=1 PRIVATE_TAG=<unset> 
PROXY_ENABLED=0 PUBLIC_KEY_COUNT=0 QUOTA_EXCEEDING=0 
SCAN_ALL_FOLDERS_DEBOUNCE_SECS=60 SECONDARY_ADDRS= SELFAVATAR=<unset> 
SHOW_EMAILS=2 SIGN_UNENCRYPTED=0 SQLITE_VERSION=3.46.1 STATS_ID=<unset> 
STATS_LAST_SENT=0 STATS_SENDING=false STD_HEADER_PROTECTION_COMPOSING= 
SYNC_MSGS=1 TEST_HOOKS= UPTIME=0h 0m 5s 
USED_TRANSPORT_SETTINGS=1: ci-c228e7@ci-chatmail.testrun.org imap:[ci-chatmail.testrun.org:993:tls:ci-c228e7@ci-chatmail.testrun.org, ci-chatmail.testrun.org:143:starttls:ci-c228e7@ci-chatmail.testrun.org, ci-chatmail.testrun.org:443:tls:ci-c228e7@ci-chatmail.testrun.org] smtp:[ci-chatmail.testrun.org:465:tls:ci-c228e7@ci-chatmail.testrun.org, ci-chatmail.testrun.org:587:starttls:ci-c228e7@ci-chatmail.testrun.org, ci-chatmail.testrun.org:443:tls:ci-c228e7@ci-chatmail.testrun.org] provider:none cert_strict 
WEBXDC_REALTIME_ENABLED=true 
--------- INBOX 3 messages ---------
Message 2 has empty body
Message 3 has empty body
Message 4 has empty body
--------- EMPTY FOLDERS: ['DeltaChat']


6.76 [MAIN-ac1] stop_ongoing
6.76 [MAIN-ac1] dc_stop_io (stop core IO scheduler)
6.76 [events-ac1] EVENT THREAD FINISHED
6.76 [MAIN-ac1] wait for event thread to finish
6.76 [MAIN-ac1] remove dc_context references, making the Account unusable
6.76 [MAIN-ac1] shutdown finished
```

May be related to 25750de4e19296aba6915cf0c11d1b047aa56c8e. I think this may be closed if we port the test to JSON-RPC and reopened if it continues to fail.

--- a/deltachat-rpc-client/tests/test_something.py
+++ b/deltachat-rpc-client/tests/test_something.py
@@ -381,6 +381,30 @@ def test_selfavatar_sync(acfactory, data, log) -> None:
     assert avatar_config != avatar_config2
 
 
+def test_dont_move_sync_msgs(acfactory, direct_imap):
+    addr, password = acfactory.get_credentials()
+    ac1 = acfactory.get_unconfigured_account()
+    ac1.set_config("bcc_self", "1")
+    ac1.set_config("fix_is_chatmail", "1")
+    ac1.add_or_update_transport({"addr": addr, "password": password})
+    ac1.bring_online()
+    ac1_direct_imap = direct_imap(ac1)
+
+    ac1_direct_imap.select_folder("Inbox")
+    # Sync messages may also be sent during configuration.
+    inbox_msg_cnt = len(ac1_direct_imap.get_all_messages())
+
+    ac1.set_config("displayname", "Alice")
+    ac1.wait_for_event(EventType.MSG_DELIVERED)
+    ac1.set_config("displayname", "Bob")
+    ac1.wait_for_event(EventType.MSG_DELIVERED)
+    ac1_direct_imap.select_folder("Inbox")
+    assert len(ac1_direct_imap.get_all_messages()) == inbox_msg_cnt + 2
+
+    ac1_direct_imap.select_folder("DeltaChat")
+    assert len(ac1_direct_imap.get_all_messages()) == 0
+
+
 def test_reaction_seen_on_another_dev(acfactory) -> None:
     alice, bob = acfactory.get_online_accounts(2)
     alice2 = alice.clone()

--- a/python/tests/test_1_online.py
+++ b/python/tests/test_1_online.py
@@ -235,25 +235,6 @@ def test_enable_mvbox_move(acfactory, lp):
     assert ac2._evtracker.wait_next_incoming_message().text == "message1"
 
 
-def test_dont_move_sync_msgs(acfactory):
-    ac1 = acfactory.new_online_configuring_account(bcc_self=True, sync_msgs=True, fix_is_chatmail=True)
-    acfactory.bring_accounts_online()
-
-    ac1.direct_imap.select_folder("Inbox")
-    # Sync messages may also be sent during the configuration.
-    inbox_msg_cnt = len(ac1.direct_imap.get_all_messages())
-
-    ac1.set_config("displayname", "Alice")
-    ac1._evtracker.get_matching("DC_EVENT_MSG_DELIVERED")
-    ac1.set_config("displayname", "Bob")
-    ac1._evtracker.get_matching("DC_EVENT_MSG_DELIVERED")
-    ac1.direct_imap.select_folder("Inbox")
-    assert len(ac1.direct_imap.get_all_messages()) == inbox_msg_cnt + 2
-
-    ac1.direct_imap.select_folder("DeltaChat")
-    assert len(ac1.direct_imap.get_all_messages()) == 0
-
-
 def test_forward_messages(acfactory, lp):
     ac1, ac2 = acfactory.get_online_accounts(2)
     chat = ac1.create_chat(ac2)

